service/gov_pay.go

```diff
@@ -11,6 +11,27 @@ import (
 	"github.com/companieshouse/payments.api.ch.gov.uk/models"
 )
 
+type GovpayResponse struct{}
```

Comment:
capital `P` in `GovPayResponse` ? seems to be the way we write it everywhere else